### DISCUSSION TITLE
Field constraints not working for File class (and possibly others)

### DISCUSSION THREAD

**Discussion by mp3cko** [fastapi/fastapi#7120] [Questions] [answered]
### Example

Here's a self-contained, [minimal, reproducible, example](https://stackoverflow.com/help/minimal-reproducible-example) with my use case:

```Python
from fastapi import FastAPI

app = FastAPI()


app.post("/create-table/{table_name}")
async def create_table(tasks : BackgroundTasks,
                       table_name: str,
                       file: UploadFile = File(...,min_length=0,max_length=1000000*5)
                       ):
```
### Description

* Boot up the API
* It errors out with the following message 

```
ValueError: On field "file" the following field constraints are set but not enforced: min_length, max_length.
For more details see https://pydantic-docs.helpmanual.io/usage/schema/#unenforced-field-constraints
```

* When `min_length` and `max_length` arguments are not present it is not erroring out

### Environment

* OS: Win10 2004
* FastAPI 0.61.1

* Python version: 3.7.9

### Additional context

I have tried reading the docs on pydantic but I don't see a solution how to fix this, also it is not mentioned anywhere in the docs

This is very important because of working with large files 

![image](https://user-images.githubusercontent.com/56165587/94370361-e854fe00-00ef-11eb-8378-491b3d7511c4.png)

**Comment by mp3cko** [comment, marked as answer]
Nevermind. I figured out it is not needed when using UploadFile...

**Comment by tiangolo** [comment]
Thanks for coming back to close the issue @maracko ☕ 

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
